lib/representable/object.rb

```diff
@@ -3,6 +3,8 @@
 
 module Representable
   module Object
```

Comment:
You should create another module Struct and leave this one intact.

Reply:
Moved everything to `Representable::Struct`, restored `Representable::Object` to its previous state.